packages/opentelemetry-plugin-mongodb/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@opentelemetry/plugin-mongodb-core",
+  "name": "@opentelemetry/plugin-mongodb",
```

Comment:
Please update the list of default supported plugins https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-node/src/config.ts#L30

Reply:
Also, please update the cache path: https://github.com/open-telemetry/opentelemetry-js/blob/master/.circleci/config.yml#L67